wp-job-manager-template.php

```diff
@@ -814,7 +823,11 @@ function get_the_job_location( $post = null ) {
 		return null;
 	}
 
-	return apply_filters( 'the_job_location', $post->_job_location, $post );
+	if ( $post->_remote_position ) {
+		return apply_filters( 'the_job_location', $post->_remote_position, $post );
+	} else {
+		return apply_filters( 'the_job_location', $post->_job_location, $post );
+	}
```

Comment:
I am not sure we need this part. I think `job_location` and `remote_position` should be separated.

Is this covering some edge case? Why this was added?

Reply:
True we probably don't. I'll revert the change since @0xDELS hasn't responded in a while. What do you think about [this](https://github.com/Automattic/WP-Job-Manager/pull/1621#pullrequestreview-564261384) comment with respect to search which right now is either 'remote' or 'location' but not both?

Reply:
@onubrooks Yes! Now that we have changes from the #2261 we can add support for multiple search locations and remote/non-remote listings

I'll push the implementation soon. 👍

Reply:
@onubrooks I pushed some updates. Now it should allow you to search multiple locations at once and also search for remote locations as well. It is a text based UI. So when you search for job located in London, you search `London`. If you search for jobs that are remote in London, then you search `london remote`.


https://user-images.githubusercontent.com/2578542/173572033-3ba12bdc-c680-41f9-a1d4-2c89acdd382a.mp4

Let me know what you think.

Reply:
This works for me!